scripts/pages/end-of-run/end-of-run.ts

```diff
@@ -29,10 +29,15 @@ class EndOfRunHandler {
 	selectedSplit: Timer.ComparisonSplit;
 
 	constructor() {
+		// End of run goes through 3 stages in sequence:
+		// 1) Run is completed after player enters end zone (EndofRun_Result_RunFinish)
+		// 2) Replay for the run has been compressed and saved to disk (EndOfRun_Result_RunSave)
+		// 3) Replay has been uploaded and verified (EndOfRun_Result_RunUpload)
+		$.RegisterForUnhandledEvent('EndOfRun_Result_RunFinish', (run) => this.onRunFinished(run));
```

Comment:
Need to declare this event in `scripts/types-mom/events.d.ts`, just search for `EndOfRun_Result_RunSave` and go based on that.

It's easy to lose track of API and events definitions otherwise (Valve's `dump_panorama_events` or whatever is very impractical and AFAIK nobody use it).

Depending on your editor it might be worth installing a TypeScript language extension, or you can run `npm install && npm run ts:watch` inside the `panorama/` directory to check via CLI.